include/bout/coordinates.hxx

```diff
@@ -35,83 +35,92 @@ class Coordinates;
 #ifndef __COORDINATES_H__
 #define __COORDINATES_H__
 
-#include "mesh.hxx"
 #include "datafile.hxx"
+#include "mesh.hxx"
 #include <bout_types.hxx>
 
 /*!
  * Represents a coordinate system, and associated operators
  *
  * This is a container for a collection of metric tensor components
- */ 
+ */
 class Coordinates {
 public:
   /// Constructor
   Coordinates(Mesh *mesh);
-  
+
   ~Coordinates() {}
-  
+
   /*!
    * Adds variables to the output file, for post-processing
-   * 
+   *
    * Must be a better way so that Coordinates doesn't depend on Datafile
    */
   void outputVars(Datafile &file);
-  
+
   Field2D dx, dy; ///< Mesh spacing in x and y
-  BoutReal dz; ///< Mesh spacing in Z
+  BoutReal dz;    ///< Mesh spacing in Z
 
   BoutReal zlength() const { return dz * nz; } ///< Length of the Z domain. Used for FFTs
 
   /// True if corrections for non-uniform mesh spacing should be included in operators
   bool non_uniform;
-  Field2D d1_dx, d1_dy;  ///< 2nd-order correction for non-uniform meshes d/di(1/dx) and d/di(1/dy)
-  
+  Field2D d1_dx,
```

Comment:
Some of the changes clang-format introduces are not improvements (in my opinion)